src/pages/Swap/index.tsx

```diff
@@ -279,14 +278,13 @@ export function Swap({
     parsedAmount,
     currencies,
     inputError: swapInputError,
+    inputTax,
+    outputTax,
   } = swapInfo
 
   const [inputTokenHasTax, outputTokenHasTax] = useMemo(
-    () => [
-      !!currencies[Field.INPUT] && !getInputTax(currencies[Field.INPUT]).equalTo(0),
-      !!currencies[Field.OUTPUT] && !getOutputTax(currencies[Field.OUTPUT]).equalTo(0),
-    ],
-    [currencies]
+    () => [!inputTax.equalTo(0), !outputTax.equalTo(0)],
+    [inputTax, outputTax]
   )
```

Comment:
i don't think you need to memoize this anymore, checking equality against zero isn't alot of work.

Reply:
discussed on slack, gonna keep the memo since sdk-core/Fraction's equalTo is a bit verbose